client/my-sites/themes/controller.js

```diff
@@ -70,7 +72,32 @@ var controller = {
 			),
 			document.getElementById( 'primary' )
 		);
+	},
+
+	details: function( context ) {
+		const user = getCurrentUser( context.store.getState() );
+
+		const Head = user
+			? require( 'layout/head' )
+			: require( 'my-sites/themes/head' );
+
+		const element = (
+			<ReduxProvider store={ context.store } >
+				<Head title="Something Theme — WordPress.com" isSheet>
+					<ThemeSheetComponent themeSlug={ context.params.slug } />
+				</Head>
+			</ReduxProvider>
+		);
+
+		// FIXME: temporary hack until we have a proper isomorphic, one tree routing solution. Do NOT do this!
+		const sheetsDomElement = document.getElementsByClassName( 'themes__sheet' )[0];
+		if ( ! sheetsDomElement ) {
```

Comment:
To avoid rendering a React tree into another React tree, we make sure that we don't have pre-existing markup — which would be the case for an SSRed page.